packages/frontend/editor-ui/src/app/composables/usePinnedData.ts

```diff
@@ -215,6 +217,8 @@ export function usePinnedData(
 			data_size: stringSizeInBytes(data.value),
 			view: displayMode,
 			run_index: runIndex,
+			workflow_id: workflowsStore.workflowId,
```

Comment:
optional - there's a test called `should trigger telemetry on successful data setting` maybe we can extend that to check the values sent are correct, not just that the event was fired